aero.minova.core.application.system.service/src/main/resources/application.properties

```diff
@@ -23,4 +21,6 @@ spring.thymeleaf.suffix=.html
 spring.http.converters.preferred-json-mapper=gson
 login_dataSource=ldap
 
-server.servlet.context-path=/cas
\ No newline at end of file
+server.servlet.context-path=/cas
```

Comment:
Bitte testen, ob das mit Firefox und der aktuellen WFC-Anwendung geht,
ohne Verschlüsselung geht.
Wenn nicht müssten würde ich es in einen anderen Pull-Request machen,
da man auch in der WFC-Anwendung was anpassen müsste.

Reply:
> Bitte testen, ob das mit Firefox und der aktuellen WFC-Anwendung geht, ohne Verschlüsselung geht. Wenn nicht müssten würde ich es in einen anderen Pull-Request machen, da man auch in der WFC-Anwendung was anpassen müsste.

Ich hab das noch mal getestet und es steht tatsächlich noch HTTP 1.1 da. Ich habe auch versucht einen Keystore für localhost einzubinden und bin vorgegangen wie hier beschrieben: https://stackoverflow.com/questions/38612704/enable-http2-with-tomcat-in-spring-boot

Ergebnis blieb gleich, HTTP 1.1.

Ich hab das Issue aus dem PullRequest raus genommen und würde es ein andermal nochmal angehen, den Branch aber jetzt erstmal mergen, da er noch Änderungen hat, nach welchen Menschen fragen.

Reply:
Dann sollte `server.http2.enabled=true` wieder entfernt werden.